x-pack/test/dataset_quality_api_integration/common/config.ts

```diff
@@ -65,6 +66,7 @@ export interface CreateTest {
   services: InheritedServices & {
     datasetQualityFtrConfig: () => DatasetQualityFtrConfig;
     registry: ({ getService }: FtrProviderContext) => ReturnType<typeof RegistryProvider>;
+    es: (context: InheritedFtrProviderContext) => Client;
```

Comment:
is this really needed? afaik you could get the `es` service by 
```
getService('es')
```

Reply:
It wasn't needed. I've removed it. Thanks for potting.